src/test/scala/org/ergoplatform/mining/ErgoMinerSpec.scala

```diff
@@ -53,6 +55,75 @@ class ErgoMinerSpec extends FlatSpec with ErgoTestHelpers with ValidBlocksGenera
     empty.copy(nodeSettings = nodeSettings, chainSettings = chainSettings)
   }
 
+  it should "not include too complex transactions" in new TestKit(ActorSystem()) {
```

Comment:
Seems this test produces unstable result

Reply:
Fixed in https://github.com/ergoplatform/ergo/pull/829/commits/e490bebd419e3de0287a4a86d8901b43ebdbdbf0